packages/components/src/components/wizard/wizard.tsx

```diff
@@ -15,28 +31,26 @@ const Wizard = ({ children, className, initial_step, onStepChange, nav, selected
         }
     }, [initial_step, getSteps]);
 
-    const onChangeStep = stats => {
+    const onChangeStep = (stats: { [key: string]: number }) => {
         // User callback
-        onStepChange(stats);
+        if (typeof onStepChange === 'function') onStepChange(stats);
```

Comment:
@likhith-deriv, since this is defined to be a function already, wouldn't it be okay to do `if (onStepChange)` or `onStepChange?.(stats)`, because it'll either be a function or `undefined`?

Reply:
Will change this as suggested